features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/GesturesDetectorWrapper.kt

```diff
@@ -24,7 +24,9 @@ internal class GesturesDetectorWrapper(
     )
 
     fun onTouchEvent(event: MotionEvent) {
-        defaultGesturesDetector.onTouchEvent(event)
+        if (defaultGesturesDetector.onTouchEvent(event)) {
```

Comment:
oh I see so returning true from the `onSingleTapUp` will fix your code here. Nice one ;)